BrainPortal/public/javascripts/userfiles.js

```diff
@@ -401,6 +401,72 @@ $(function() {
 
     /* Show/Hide dynamic actions/menu elements according to current selection */
     (function () {
+      function launch_task() {
+        var parameters    = window.location.search.split(/\?|&/);
```

Comment:
There must exist query parameter APIs or helpers functions, at least in jQuery ? Also, given the page is rendered as a consequence of a GET, instead of duplicating all this code from userfiles/launch_task.js, why not simply trigger the behavior with one synthetic event?